fixtures/proc-macro/src/lib.rs

```diff
@@ -304,4 +304,29 @@ pub fn join(parts: &[String], sep: &str) -> String {
     parts.join(sep)
 }
 
+// Custom names
+#[derive(uniffi::Object)]
+pub struct Renamed;
+
+// `renamed_new` becomes the default constructor because it's named `new`
+#[uniffi::export(name=new)]
+impl Renamed {
+    #[uniffi::constructor]
+    fn renamed_new() -> Arc<Self> {
+        Arc::new(Self)
+    }
+}
+
+#[uniffi::export(name=func)]
+impl Renamed {
+    fn renamed_func(&self) -> bool {
+        true
+    }
+}
```

Comment:
How about another dummy attribute like constructor (just to have a place to attach method attributes) instead of using one-item impls?


```suggestion
#[uniffi::export]
impl Renamed {
    #[uniffi::method(name = func)]
    fn renamed_func(&self) -> bool {
        true
    }
}
```